src/main/java/org/datadog/jenkins/plugins/datadog/listeners/DatadogGraphListener.java

```diff
@@ -173,4 +180,35 @@ long getTime(FlowNode startNode, FlowNode endNode) {
         }
         return 0;
     }
+    
+    String getResultTag(@Nonnull FlowNode endNode) {
```

Comment:
This could potentially live in `BuildData` and we could check for instance of `FlowNode` in `getResult()`